### PR TITLE
Footer section

### PR DESCRIPTION
**Pull request by PabloFLPs**
Just creating a Footer section.

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        icon: (props: IIcon) => (
+        <svg fill="currentColor" viewBox="0 0 24 24" {...props}>
+            <path d="M8.29 20.251c7.547 0 11.675-6.253 11.675-11.675 0-.178 0-.355-.012-.53A8.348 8.348 0 0022 5.92a8.19 8.19 0 01-2.357.646 4.118 4.118 0 001.804-2.27 8.224 8.224 0 01-2.605.996 4.107 4.107 0 00-6.993 3.743 11.65 11.65 0 01-8.457-4.287 4.106 4.106 0 001.27 5.477A4.072 4.072 0 012.8 9.713v.052a4.105 4.105 0 003.292 4.022 4.095 4.095 0 01-1.853.07 4.108 4.108 0 003.834 2.85A8.233 8.233 0 012 18.407a11.616 11.616 0 006.29 1.84" />
+        </svg>
+        ),
+    },
+    {
+        name: 'GitHub',
+        href: 'https://github.com/PabloFLPs',
+        icon: (props: IIcon) => (
```
Comment (Erickinhou) [inline src/components/Footer.tsx]: Props can handle more than IIcon interface, like events and other HTML properties. I think you can use React.HTMLAttributes or something like this to type your icon props.

**Review by Erickinhou — CHANGES_REQUESTED** [review]
Works but I think it's better improve a little bit the typing :)

File: src/components/Footer.tsx (hunk 1 above)
Comment (PabloFLPs) [reply src/components/Footer.tsx]: Oh, very cool. I need just to extend my interface like the example below?

image

**Review by Erickinhou — APPROVED** [review]

File: src/components/Footer.tsx (hunk 1 above)
Comment (Erickinhou) [reply src/components/Footer.tsx]: yes, something like this.

**Pull request merged**